src/lib.rs

```diff
@@ -78,16 +83,17 @@ impl error::Error for Error {
 /// ```
 #[derive(Debug)]
 pub struct RedisConnectionManager {
-    connection_info: redis::ConnectionInfo
+    connection_info: redis::ConnectionInfo,
 }
 
 impl RedisConnectionManager {
     /// Creates a new `RedisConnectionManager`.
     ///
     /// See `redis::Client::open` for a description of the parameter
     /// types.
-    pub fn new<T: redis::IntoConnectionInfo>(params: T)
-            -> Result<RedisConnectionManager, redis::RedisError> {
+    pub fn new<T: redis::IntoConnectionInfo>(
+        params: T,
+    ) -> Result<RedisConnectionManager, redis::RedisError> {
```

Comment:
Was this auto-formatted?

Reply:
yes, I have my editor configured to run rustfmt on save. I can edit the commit to exclude autoformatting if that's preferable.

Reply:
Nope, fine with me (not the biggest fan of how it looks for this method, but tooling over my own preference!)